### ISSUE TITLE
Feature Suggestion: Import Avatars from MakeHuman and others; 

### ISSUE THREAD

**Issue by ADimensionBeyond** [adlnet/Sandbox#248]
Import avatar rigging from MakeHuman- (and other sources) That might be a really good way to make custom avatars. Also useful would be a simple drag and drop system to dress and change outfits, hair etc. like in other virtual world systems.

**Comment by rchadwic** [comment]
We've just had Jeff rebuild our avatars based on the Mixamo avatar base. I'm not sure if this is the same thing, but it might work for you. I think he has written an article on how to export from Mixamo... I"ll post the link when it's up.

**Comment by wilo1** [comment]
I would love to see this article on mixamo export!

**Comment by rchadwic** [comment]
Yeah, our current avatar is actual the default Mixamo one. I'm not sure what sort of process Jeff had to go through to get it in.

**Comment by wilo1** [comment]
We can get mixamo avatars in but unless it's keyframe you can only have a very basic walk animation when it's exported as collada? Unless I am totally missing something?

**Comment by rchadwic** [comment]
Yeah I have no idea. I know Jeff had to jump through some hoops, but I don't know what they were. Can Mixamo export a single sequence of movement by stringing together different mo-cap files?
